--- dmenu-orig/config.def.h	2023-09-09 16:56:16.722074905 +0100
+++ dmenu-new/config.def.h	2023-09-09 17:06:39.865410267 +0100
@@ -7,11 +7,12 @@
 	"monospace:size=10"
 };
 static const char *prompt      = NULL;      /* -p  option; prompt to the left of input field */
+/* Dracula theme */
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
-	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
-	[SchemeOut] = { "#000000", "#00ffff" },
+	[SchemeNorm] = { "#F8F8F2", "#282A36" },
+	[SchemeSel] = { "#E6E6E6", "#BD93F9" },
+	[SchemeOut] = { "#4D4D4D", "#FF79C6" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
 static unsigned int lines      = 0;
