diff --color -u a/config.def.h b/config.def.h
--- a/config.def.h	2022-10-04 18:38:18.000000000 +0100
+++ b/config.def.h	2023-09-08 18:03:10.456568794 +0100
@@ -7,15 +7,16 @@
 static const int topbar             = 1;        /* 0 means bottom bar */
 static const char *fonts[]          = { "monospace:size=10" };
 static const char dmenufont[]       = "monospace:size=10";
-static const char col_gray1[]       = "#222222";
-static const char col_gray2[]       = "#444444";
-static const char col_gray3[]       = "#bbbbbb";
-static const char col_gray4[]       = "#eeeeee";
-static const char col_cyan[]        = "#005577";
+/* Dracula theme */
+static const char dark_grey[]       = "#282A36";
+static const char medium_grey[]     = "#4D4D4D";
+static const char light_grey[]      = "#BFBFBF";
+static const char white[]           = "#E6E6E6";
+static const char purple[]          = "#BD93F9";
 static const char *colors[][3]      = {
-	/*               fg         bg         border   */
-	[SchemeNorm] = { col_gray3, col_gray1, col_gray2 },
-	[SchemeSel]  = { col_gray4, col_cyan,  col_cyan  },
+    /*               fg         bg         border   */
+    [SchemeNorm] = { light_grey,dark_grey, medium_grey },
+    [SchemeSel]  = { white,     purple,    purple  },
 };

 /* tagging */
@@ -56,7 +57,7 @@
 #define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }

 /* commands */
-static const char *dmenucmd[] = { "dmenu_run", "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run", "-fn", dmenufont, "-nb", dark_grey, "-nf", light_grey, "-sb", purple, "-sf", white, NULL };
 static const char *termcmd[]  = { "st", NULL };

 static const Key keys[] = {
